buildfactories.py

```diff
@@ -20,21 +20,26 @@ def get_cmake_step(link, type, frameworks = 'no'):
     build_frameworks = ''
     suffix = ''
 
-    if frameworks == 'frameworks':
+    if 'frameworks' in osxExtra:
         build_frameworks += '-DSFML_BUILD_FRAMEWORKS=TRUE'
         suffix = [link, type, 'frameworks'];
     else:
         build_frameworks += '-DSFML_BUILD_FRAMEWORKS=FALSE'
         suffix = [link, type];
 
+    if 'oldSDK' in osxExtra:
+        build_sdk += '-DCMAKE_OSX_DEPLOYMENT_TARGET=10.7'
+        build_sdk += '-DCMAKE_OSX_SYSROOT=/Developer/SDKs/MacOSX10.7.sdk'
```

Comment:
I'm not sure this is OK w.r.t. how it's used in `command` below. Should it be two separate variables?